src/e2e-test/rule-test/e2e-test.spec.js

```diff
@@ -7,10 +7,10 @@ const supertest = require('supertest');
 const express = require('express');
 var fs = require("fs");
 var path  = require('path');
-var constants = require('../lib/constants/constants');
+var constants = require('../../lib/constants/constants');
 const API_KEY_HEADER = "X-MS-CONVERSION-API-KEY";
 const apiKeys = ['123', '456'];
-var app = require("../routes")(express());
+var app = require("../../routes")(express());
 
 // load testcases
```

Comment:
How about changing the file name to  `rule-test.spec.js` ? In addition, can you modify the file structure of rule test to be consistent with expression test? For example, add `util` folder and change the folder name `test-sample`.

Reply:
Fixed yet.